ludwig/features/date_feature.py

```diff
@@ -164,6 +162,11 @@ def update_config_with_metadata(
     ):
         pass
 
+    def create_input(self):
```

Comment:
Maybe we can rename this `create_sample_input` to clarify that this is used for Torchscript to trace the graph.